src/utils/runtimeConfig.ts

```
@@ -0,0 +1,38 @@
+import { prisma } from "./prisma";
```

ESLint

[error] Replace `{·prisma·}·from·"./prisma` with `logger·from·"./logger`

(prettier/prettier)